ocaml/lambda/translobj.ml

```diff
@@ -186,7 +186,7 @@ let oo_wrap env req f x =
                         [lambda_unit; lambda_unit; lambda_unit],
                         Loc_unknown)
                 in
-                Llet(StrictOpt, Pgenval, id,
+                Llet(StrictOpt, Lambda.layout_class, id,
```

Comment:
I don't think this is actually a class. No idea what it is though.

Reply:
```suggestion
                Llet(StrictOpt, Lambda.layout_block, id,
```